### ISSUE TITLE
Build error: MediationNativeListenerImplementor is not abstract and does not override abstract method

### ISSUE THREAD

**Issue by Mataboge** [xamarin/GooglePlayServicesComponents#369]
### Xamarin.Android Version (eg: 6.0):
Build Target Android 10

### Operating System & Version (eg: Mac OSX 10.11):
Windows 10
Viisul Studio 19 16.6.3

### Google Play Services Version
19.1.0-preview01

*   [] GPS-FB with AndroidX dependencies (versions `1xx.yyyy.zz`)
    

*   [] GPS-FB with Android.Support (legacy) dependencies (versions `71.yyyy.zz`)

### Describe your Issue

The preview version gives this error during build:
```
2>  obj\Debug\100\android\src\mono\com\google\android\gms\ads\mediation\MediationNativeListenerImplementor.java:4: error: MediationNativeListenerImplementor is not abstract and does not override abstract method zza(MediationNativeAdapter,NativeCustomTemplateAd,String) in MediationNativeListener
2>  public class MediationNativeListenerImplementor
2>         ^
2>  Note: Some input files use or override a deprecated API.
2>  Note: Recompile with -Xlint:deprecation for details.
2>  Note: Some input files use unchecked or unsafe operations.
2>  Note: Recompile with -Xlint:unchecked for details.
2>  1 error
2> \obj\Debug\100\android\src\mono\com\google\android\gms\ads\mediation\MediationNativeListenerImplementor.java(4,8): javac.exe error JAVAC0000:  error: MediationNativeListenerImplementor is not abstract and does not override abstract method zza(MediationNativeAdapter,NativeCustomTemplateAd,String) in MediationNativeListener
2>\obj\Debug\100\android\src\mono\com\google\android\gms\ads\mediation\MediationNativeListenerImplementor.java(4,8): javac.exe error JAVAC0000: public class MediationNativeListenerImplementor
2>\obj\Debug\100\android\src\mono\com\google\android\gms\ads\mediation\MediationNativeListenerImplementor.java(4,8): javac.exe error JAVAC0000: 
```

The latest Admob android release notes also speak of a way of disabling Mediation adapter on their version 19.2.0 and this method is not implemented.

https://developers.google.com/admob/android/rel-notes

### Relevant information

Packages used:

```
    <PackageReference Include="Xamarin.Forms" Version="4.7.0.1080" />
    <PackageReference Include="Xamarin.Essentials" Version="1.5.3.2" />
<PackageReference Include="Xamarin.AndroidX.Annotation">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Core">
      <Version>1.2.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.UI">
      <Version>1.0.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.Utils">
      <Version>1.0.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Browser">
      <Version>1.2.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Google.Android.Material">
      <Version>1.1.0.1-rc3</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Fragment">
      <Version>1.2.4.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Media">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Firebase.Ads">
      <Version>119.1.0-preview01</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4">
      <Version>1.0.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.AppCompat">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.CardView">
      <Version>1.0.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.MediaRouter">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.VectorDrawable">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.AppCompat.Resources">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.MediaRouter">
      <Version>1.1.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.WebKit">
      <Version>1.2.0.1</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.AndroidX.Migration" Version="1.0.6.1" />
```

Build settings (tools)

```
<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <DebugSymbols>true</DebugSymbols>
    <DebugType>full</DebugType>
    <Optimize>false</Optimize>
    <OutputPath>bin\Debug</OutputPath>
    <DefineConstants>DEBUG;</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <AndroidLinkMode>None</AndroidLinkMode>
    <AotAssemblies>false</AotAssemblies>
    <EnableLLVM>false</EnableLLVM>
    <BundleAssemblies>false</BundleAssemblies>
    <AndroidUseSharedRuntime>true</AndroidUseSharedRuntime>
    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
    <AndroidSupportedAbis>armeabi-v7a;x86;x86_64</AndroidSupportedAbis>
    <AndroidCreatePackagePerAbi>false</AndroidCreatePackagePerAbi>
    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
    <AndroidLinkSkip />
    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
    <AndroidDexTool>d8</AndroidDexTool>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <DebugSymbols>false</DebugSymbols>
    <DebugType>pdbonly</DebugType>
    <Optimize>true</Optimize>
    <OutputPath>bin\Release</OutputPath>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <AndroidManagedSymbols>true</AndroidManagedSymbols>
    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
    <AotAssemblies>false</AotAssemblies>
    <EnableLLVM>false</EnableLLVM>
    <BundleAssemblies>false</BundleAssemblies>
    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a</AndroidSupportedAbis>
    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
    <AndroidLinkTool>r8</AndroidLinkTool>
    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
    <AndroidCreatePackagePerAbi>false</AndroidCreatePackagePerAbi>
    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
    <AndroidPackageFormat>aab</AndroidPackageFormat>
    <AndroidUseAapt2>true</AndroidUseAapt2>
    <AndroidDexTool>d8</AndroidDexTool>
    <AndroidLinkSkip>Xamarin.GooglePlayServices.Ads.Lite</AndroidLinkSkip>
  </PropertyGroup>
```

### Steps to Reproduce (with link to sample solution if possible):

I Just installed the Preview package and build the solution. Tried all variants of the google ad packages, ie
xamarin.googlepalyservices.ads
xamarin.googlepalyservices.ads.lite
xamarin.Firebase.ads
xamarin.Firebase.ads.lite

they all behave the same

**Comment by moljac** [comment]
Related/Duplicate

https://github.com/xamarin/GooglePlayServicesComponents/issues/347

**Comment by moljac** [comment]
Must see whether this might be a regression. It has been solved.

**Comment by moljac** [comment]
119.1.0-preview02 published

 
https://www.nuget.org/packages/Xamarin.GooglePlayServices.Ads.Lite/119.1.0-preview02

@Mataboge 
Could you please provide feedback about this published package, please? Even better close this issue yourself.

Thanks.

**Comment by Mataboge** [comment]
The published 119.1.0-preview02 solved the issue. Thank you
